scripts/build_components.sh

```diff
@@ -20,7 +20,7 @@ function usage {
 
 # Parse the arguments
 while [[ "$#" -gt 0 ]]; do case $1 in
-  -r |--registry) registry="$2"; shift;;
+  -rg|--registry) registry="$2"; shift;;
```

Comment:
Is `-r` still used somewhere?

Reply:
I don't think so. It was duplicate before, since it's used for `--repo` as well.